ModelLibrary/Model/MOrderLine.cs

```diff
@@ -4506,6 +4506,26 @@ protected override bool BeforeSave(bool newRecord)
                         margin = Decimal.Round(Decimal.Subtract(sp, pp), GetPrecision(), MidpointRounding.AwayFromZero);
                         marginper = Decimal.Round(Decimal.Multiply(Decimal.Divide(margin, sp)
                             , Env.ONEHUNDRED), GetPrecision(), MidpointRounding.AwayFromZero);
+
+
+                        SetLineNetAmt(Decimal.Round(sp, 2));
+                        decimal taxAmt;
+                        decimal lineTotalAmt;
+
+                        MOrderTax tax = MOrderTax.Get(this, GetPrecision(), false, Get_TrxName());
+                        MTax taxRate = tax.GetTax();
+                        if (taxRate.GetRate() != 0)
+                        {
+                            taxAmt = ((GetPriceEntered() * GetQtyEntered()) * (taxRate.GetRate() / 100) * Util.GetValueOfDecimal(duration)) / 12;
+                        }
+                        else
+                        {
+                            taxAmt = 0;
+                        }
+                        SetTaxAmt(Decimal.Round(taxAmt, 2));
+                        lineTotalAmt = sp + taxAmt;
+
+                        SetLineTotalAmt(Decimal.Round(lineTotalAmt, 2));
```

Comment:
Do not fix rounding i.e. 2, get it from currency like used above with the function getPrecision